providers/sample/src/java/org/sakaiproject/provider/user/SampleCandidateDetailProvider.java

```diff
@@ -64,7 +65,7 @@ public class SampleCandidateDetailProvider implements CandidateDetailProvider
 	private final static String SYSTEM_PROP_DISPLAY_ADDITIONAL_INFORMATION = "displayAdditionalInformation";
 	private final static String SYSTEM_PROP_DISPLAY_SPECIAL_NEEDS = "displaySpecialNeeds";
 	private final static String SYSTEM_PROP_USE_INSTITUTIONAL_NUMERIC_ID = "useInstitutionalNumericID";
-	private final static String SYSTEM_PROP_ENCRYPT_NUMERIC_ID = "encryptInstitutionalNumericID";
+	private final static String SYSTEM_PROP_ENCRYPT_CANDIDATE_DETAILS = "encryptCandidateDetails";
```

Comment:
Umm, I'd like to respect the current property, maybe we can chain them.

serverConfigurationService.getBoolean(SYSTEM_PROP_ENCRYPT_NUMERIC_ID, serverConfigurationService.getBoolean(SYSTEM_PROP_ENCRYPT_CANDIDATE_DETAILS , true))

Reply:
If any institution has this property, use the value of that institution instead of the new one. WDYT?

Reply:
Okay, makes sense, but just for the student number, right? Then we will have the same behavior.

Reply:
Yep